app/main.js

```diff
@@ -34,6 +34,15 @@ import User from './user';
     window.appState = state;
     window.appEmit = emitter.emit.bind(emitter);
     let unsupportedReason = null;
+
+    emitter.on('render', () => {
+      if (state.animation) {
```

Comment:
this is an experiment. i removed the only animation that was using this so it does nothing right now